addons/l10n_fr_pos_cert/models/res_company.py

```diff
@@ -65,32 +64,31 @@ def build_order_info(order):
 
             if not orders:
                 msg_alert = (_('There isn\'t any order flagged for data inalterability yet for the company %s. This mechanism only runs for point of sale orders generated after the installation of the module France - Certification CGI 286 I-3 bis. - POS') % self.env.company.name)
-                hash_verified = False
 
             previous_hash = u''
-            start_order_info = []
+            corrupted_orders = []
             for order in orders:
                 if order.l10n_fr_hash != order._compute_hash(previous_hash=previous_hash):
-                    msg_alert = (_('Corrupted data on point of sale order with id %s.') % order.id)
-                    hash_verified = False
-                    break
+                    corrupted_orders.append(order.name)
+                    msg_alert = (_('Corrupted data on point of sale order'))
```

Comment:
Don't change the msg_alert. As we are in stable, you will loose the translation (french one), if the module is updated with your modification in the view, msg_alert is not used in it anymore, if it is not, it'll at least show the last corrupted order.

Reply:
@pimodoo Thanks for your feedback.
I edited it.